packages/gafl-webapp-service/src/pages/multibuy/remove-licence/__tests__/route.spec.js

```diff
@@ -0,0 +1,139 @@
+import { getData, validator } from '../route.js'
+import { createMockRequest } from '../../../../__mocks__/request.js'
+import pageRoute from '../../../../routes/page-route.js'
+import { nextPage } from '../../../../routes/next-page.js'
+import { VIEW_LICENCES } from '../../../../uri.js'
+import { licenceTypeDisplay, licenceTypeAndLengthDisplay } from '../../../../processors/licence-type-display.js'
+import { displayStartTime } from '../../../../processors/date-and-time-display.js'
+import { addLanguageCodeToUri } from '../../../../processors/uri-helper.js'
+
+jest.mock('../../../../processors/uri-helper.js', () => ({
+  addLanguageCodeToUri: jest.fn((_request, uri) => uri)
+}))
+jest.mock('../../../../processors/licence-type-display.js')
+jest.mock('../../../../processors/date-and-time-display.js')
+jest.mock('../../../../routes/page-route.js')
+
+const permission = {
+  licensee: {
+    firstName: 'Turanga',
+    lastName: 'Leela'
+  },
+  licenceType: 'trout-and-coarse',
+  numberOfRods: '2',
+  licenceLength: '8D',
+  permit: { cost: 12 }
+}
+
+const catalog = Symbol('mock catalog')
+
+describe('remove-licence > default', () => {
+  it('should call the pageRoute with remove-licence, /buy/remove-licence, validator and nextPage', async () => {
+    expect(pageRoute).toBeCalledWith('remove-licence', '/buy/remove-licence', validator, nextPage, getData)
+  })
+})
+
+describe('remove-licence > getData', () => {
+  const mockRequest = createMockRequest({
```

Comment:
Should we be using this createMockRequest function? I'm concerned about readability and inconsistency of this not being done throughout. Might be one to discuss in a 3 amigos?